WcaOnRails/config/locales/en.yml

```diff
@@ -854,6 +863,13 @@ en:
         information_html: "This is a reminder that registration for <a href=%{competition_url}>%{competition}</a> opens in less than 24 hours."
         not_accepted_html: "You have registered but your registration has not been accepted."
         more_info_html: "Please see the <a href=%{competition_url}>competition website</a> for more information and registration requirements."
```

Comment:
Also, I was following the pattern here previously. Is it worth it to also update these?

Reply:
To be honest we will have to change the key name (because if we just change the variable name it will cause issues for existing translations which would exist with the other variable name), and this means translating the same thing again for translators.
So I actually don't know if it's worth :s

I'd say just leave it there since it's not broken.